fuse_examples/classification/knight/baseline/dataset.py

```diff
@@ -121,19 +121,26 @@ def knight_dataset(data_dir: str = 'data', cache_dir: str = 'cache', split: dict
     else: # split can contain BOTH 'train' and 'val', or JUST 'test'
         image_dir = os.path.join(data_dir, 'images')
         json_filepath = os.path.join(data_dir, 'features.json')
-        if test_labels:
+        if only_labels:
             json_labels_filepath = os.path.join(data_dir, 'knight_test_labels.json') 
             gt_processors = {
                 'gt_global': KiCGTProcessor(json_filename=json_labels_filepath, columns_to_tensor={'task_1_label':torch.long, 'task_2_label':torch.long}, test_labels=True)
             }
         else:
             gt_processors = {}
-    # we use the same processor for the clinical data and ground truth, since both are in the .csv file
-    # need to make sure to discard the label column from the data when using it as input
-    input_processors = {
-        'image': KiTSBasicInputProcessor(input_data=image_dir, resize_to=resize_to),
-        'clinical': KiCClinicalProcessor(json_filename=json_filepath)
-    }
+
+    if only_labels:
```

Comment:
how will this support the "test" mode where we do have the test labels but also want to predict (i.e use the images)?
maybe we should have two parameters? test_labels AND labels_only?

Reply:
We don't support it - and we didn't support it in the previous version of the code as well.
images only mode is for inference and labels only mode is for evaluation.
Does it seem important to you to support?